ddtrace/internal/telemetry/data/host.py

```diff
@@ -0,0 +1,48 @@
+import platform
+from typing import Optional
+from typing import TypedDict
+
+from ...hostname import get_hostname
+from ...runtime.container import get_container_info
+
+
+Host = TypedDict(
+    "Host",
+    {
+        "os": str,
+        "hostname": str,
+        "os_version": str,
+        "kernel_name": str,
+        "kernel_release": str,
+        "kernel_version": str,
+        "container_id": str,
+    },
+)
+
+_host = None  # type: Optional[Host]
+
+
+def get_host():
+    global _host
+    if _host:
+        return _host
+
+    _host = {
+        "os": "",
+        "hostname": get_hostname(),
+        "os_version": "",
```

Comment:
we should be able to get this from `platform`

Reply:
I believe we are only able to get os information from `platform` and not kernel version/name/release. So the host typeddict should be set like this:

```
HOST = {
    "os": platform.system(),
    "hostname": get_hostname(),
    "os_version": platform.version(),
    "kernel_name": "",
    "kernel_release": "",
    "kernel_version": "",
    "container_id": get_containter_id(),
}
```